mergin/client.py

```diff
@@ -38,9 +38,8 @@ def decode_token_data(token):
 
 
 class MerginClient:
-    def __init__(self, url=None, auth_token=None, login=None, password=None, plugin_version=None):
+    def __init__(self, url=None, auth_token=None, login=None, password=None, plugin_version=None, proxy_config=None):
```

Comment:
Could you please add a docstring for expected proxy_config? (and ideally also for other parameters...)